Mathlib/CategoryTheory/Presentable/AdjointFunctorTheorems.lean

```diff
@@ -0,0 +1,59 @@
+module
+
+public import Mathlib.CategoryTheory.Adjunction.AdjointFunctorTheorems
+public import Mathlib.CategoryTheory.Presentable.StrongGenerator
+public import Mathlib.CategoryTheory.Presentable.LocallyPresentable
+
+universe v u
+
+open CategoryTheory Limits Functor
+
+namespace CategoryTheory.Adjunction
+
+variable {C D : Type u} [Category.{v} C] [Category.{v} D] [IsLocallyPresentable.{v} C]
+    [IsLocallyPresentable.{v} D]
+
+lemma presentableAdjointFunctorTheorem₁ (L : C ⥤ D) [PreservesColimits L] [HasColimits C] :
+    L.IsLeftAdjoint := by
+  sorry
+
+lemma presentableAdjointFunctorTheorem₂ (R : C ⥤ D) [IsAccessible.{v} R] [PreservesLimits R]
+    [HasLimits C] : R.IsRightAdjoint := by
```

Comment:
Could you prove the result under the same assumptions on universes as for most of the `IsLocallyPresentable` API, i.e. using `IsLocallyPresentable.{w}` assumptions? (and suitable `PreservesLimitsOfSize`, etc.). As a locally `w`-presentable category has to be `w`-small, the "more general" case I am suggesting should follow easily from your version. One question is whether the generalization should be done here or in the `Adjunction.AdjointFunctorTheorems` file...